### ISSUE TITLE
Linter: Implement `svg-has-accessible-text` rule

### ISSUE THREAD

**Issue by marcoroth** [marcoroth/herb#1226]
Implement the [`GitHub::Accessibility::SvgHasAccessibleText`](https://github.com/github/erblint-github/blob/main/lib/erblint-github/linters/github/accessibility/svg_has_accessible_text.rb) rule from [`erblint-github`](https://github.com/github/erblint-github).

### Rule Details

`<svg>` must have accessible text. Set `aria-label`, or `aria-labelledby`, or nest a `<title>` element. However, if the `<svg>` is purely decorative, hide it with `aria-hidden="true"`.

### Examples

**Incorrect** code for this rule:

```html
<svg height="100" width="100">
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"/>
</svg>
```

**Correct** code for this rule:

```html
<svg aria-label="A circle" height="100" width="100">
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"/>
</svg>

<svg aria-labelledby="circle_title" height="100" width="100">
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"/>
</svg>

<svg height="100" width="100">
  <title>A circle</title>
  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"/>
</svg>
```

### References

- [erblint-github source](https://github.com/github/erblint-github/blob/main/lib/erblint-github/linters/github/accessibility/svg_has_accessible_text.rb)
- [erblint-github docs](https://github.com/github/erblint-github/blob/main/docs/rules/accessibility/svg-has-accessible-text.md)
- [Accessible SVGs (CSS-Tricks)](https://css-tricks.com/accessible-svgs/)

### Parent Issue

- #1205

**Comment by joelhawksley** [comment]
I'll take this one! PR: https://github.com/marcoroth/herb/pull/1669
